test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -270,6 +285,12 @@ describe('33acrossBidAdapter:', function () {
       return this;
     }
 
+    this.withUserIds = (eids) => {
+      bidRequests[0].userIdAsEids = eids;
```

Comment:
Curious at to why we have `bidRequests` as an array in our tests?

Reply:
that is how it is passed to the adapter by prebid core. The array represents each ad unit on page that is configured with us to bid on.